WebDevStudios/WDS-Blocks
# Feature/#56 support 3.4.0

Closes #55, #56

### DESCRIPTION ###
1. Updates `UrlInput` as it has been deprecated in favor of `URLInput`.
2. Updates `decodeEntities` import. It is no longer part of `wp.utils`—now part of `wp.htmlEntities`.
3. Fixes issue with the update that throws errors during activation—documented here #55.

### OTHER ###
- [X] Does this issue pass linting? (ESLint)

### STEPS TO VERIFY ###
- Checkout branch.
- Make sure 3.4.0 Gutenberg is installed.
- Test Activation (should be successful if Gutenberg is active, or error if not).
- Add/modify Blocks.

### NOTES ###
It would appear that several core Gutenberg blocks are throwing react deprecation errors which are beyond the scope of this particular plugin, but also shouldn't affect the performance of the plugin.

May need to turn `debug` off.

The first two fixes will error below 3.4.0 because of the depreciation, but the third fix works on 3.2.0+ as far as I can tell. We haven't really been worrying about that, but not sure if it would be possible to throw an error
